db/version_edit.cc

```diff
@@ -217,22 +217,30 @@ bool VersionEdit::EncodeTo(std::string* dst) const {
 
   for (const auto& blob_file_addition : blob_file_additions_) {
     PutVarint32(dst, kBlobFileAddition);
-    blob_file_addition.EncodeTo(dst);
+    std::string encoded;
```

Comment:
@ltamasi This breaks backward compatibility. But without doing so, users of BlobDB cannot downgrade. What do you think?

Reply:
I'll remove changes related to BlobDB from this PR to make it focus on WAL edits.

Reply:
> @ltamasi This breaks backward compatibility. But without doing so, users of BlobDB cannot downgrade. What do you think?

Nice catch, @cheng-chang and thanks for letting me know! In hindsight, the blob file related tags shouldn't have been put in the "ignorable" range since if you have blob files and downgrade to a version that cannot read these, you won't be able to read your data. Fortunately, since these are related to the integrated BlobDB project (which is WIP), we can still move these to the "non-ignorable" range. I'll take care of that.